Cargo.toml

```diff
@@ -20,4 +20,5 @@ members = [
     "standby",
     "twilight",
     "util",
+    "gateway-queue",
```

Comment:
This should be `gateway/queue` since it's a sub-crate of the gateway, similar to the intended meaning of the layout of the `cache` directory. And this should also be alphabetical